tests/infra/network.py

```diff
@@ -635,6 +637,8 @@ def join_node(
     def trust_node(self, node, args, valid_from=None, validity_period_days=None):
         primary, _ = self.find_primary()
         try:
+            # Note: the time it takes for a learner to catch up is proportional
+            # to the ledger size; we may want to factor that into the timeout.
```

Comment:
There's already an argument for this, `--ledger-recovery-timeout`. Perhaps we want to rename it and use it here?

Reply:
That's great, thanks! Maybe `ledger-read-timeout`?